aries_cloudagent/anoncreds/models/anoncreds_revocation.py

```diff
@@ -116,13 +131,17 @@ class Meta:
     issuer_id = fields.Str(
         description="Issuer Identifier of the credential definition or schema",
         data_key="issuerId",
+        example=INDY_OR_KEY_DID_EXAMPLE,
```

Comment:
I'm a little hesitant about the only examples showing as old Indy values. Is it possible to provide more than one example so we could do something like `did:example:123`?

Also, I believe most of these fields should be in the `metadata` kwarg now after updates to marshmallow, as a side note. That change probably belongs in a separate PR.

Reply:
I had thought about that. I was thinking when we add more methods we could figure something out for the other examples. For the metadata stuff I saw both methods used and wasn't sure the correct way.